sources/manager_wakatime.py

```
@@ -0,0 +1,127 @@
+    @classmethod
+    def get_api_key(cls) -> Optional[str]:
+        """
+        Get the WakaTime API key from environment.
+
+        Returns:
+            The API key or None if not set.
+        """
+        return EM.WAKATIME_API_KEY if EM.WAKATIME_API_KEY else None
+
+    @classmethod
+    async def init(cls):
```

Ruff (0.14.10)

Missing return type annotation for classmethod `init`

Add return type annotation: `None`

(ANN206)